### DISCUSSION TITLE
Staggered animation in Reanimated 2 Paradigm?

### DISCUSSION THREAD

**Discussion by kiddyfurby** [software-mansion/react-native-reanimated#1736] [Q&A]
Similar to https://github.com/software-mansion/react-native-reanimated/issues/540,
What would one do to run parallel animations that stagger from each other with a fixed time interval?

I tried this:
setTimeout(() => { a.value = withRepeat(withTiming(1, d2), -1 }, 500);
setTimeout(() => { b.value = withRepeat(withTiming(1, d2), -1}, 1000);
...

With that, b nicely delay itself behind a by 500 ms,
BUT when the device resume from sleep (screen off and on), a and b became synchronized, the 500 ms stagger is gone.

**Comment by Szymon20000** [comment]
I thnik that there are planty of ways. Here is one of them: `withRepeat(withSequence(withTiming(1, config), withDelay(500, withTiming(100, config))), -1)`
